content/cli/orch_cli/deployments.md

```diff
@@ -500,6 +500,192 @@ Delete labels from a specific deployment.
+`NAME` is the name of the deployment schedule.
+
+##### Optional flags
+All the regular optional flags available for `list` commands, in addition to:
+
+*  `--preview INTEGER`   -  Preview the N next dates for the workflow execution to run.
+
+
+#### schedule create
+
+##### Usage
+
+`cfy deployments schedule create [OPTIONS] WORKFLOW_ID`
+
+Schedule the execution of a workflow on a given deployment.
+
+`WORKFLOW_ID` is the name of the workflow.
+
+##### Mandatory flags
+
+*  `-d, --deployment-id TEXT`   -  The unique identifier for the deployment.
+*  `-s, --since TEXT`           -  The earliest possible time to run. Supported formats:
+                                   `YYYY-MM-DD HH:MM`, `HH:MM`, or a relative time expression such as `+2 weeks` or `+1day+10min`.
+
+##### Optional flags
+
+*  `-n, --schedule-name TEXT`   -  A name for the schedule. If not provided, defaults to `{deployment-id}_{workflow-id}`.
+*  `-p, --parameters TEXT`      -  Parameters for the workflow 
+                                   (Can be provided as wildcard based paths (*.yaml, /my_inputs/ etc.) to YAML files, a JSON string or as `key1=value1;key2=value2`). This argument can be used multiple times.
+*  `--allow-custom-parameters`  -  Allow passing custom parameters (which were not defined in the workflow's schema in the blueprint) to the execution.
+*  `-f, --force`                -  Execute the workflow even if there is an ongoing execution for the given deployment.
+*  `--dry-run`                  -  If set, no actual operations will be performed. This only prints the executed tasks, without side effects.
+*  `--wait-after-fail INTEGER`  -  When a task fails, wait this many seconds for already-running tasks to return [default: 600].
+*  `-u, --until TEXT`           -  The latest possible time to run. Supported formats: 
+                                   `YYYY-MM-DD HH:MM`, `HH:MM`, or a relative time expression such as `+2 weeks` or `+1day+10min`.
+*  `--tz TEXT`                  -  The timezone to be used for scheduling, e.g. `EST` or `Asia/Jerusalem`. By default, the local timezone will be used. 
+                                   Supports any timezone in the [tz database](en.wikipedia.org/wiki/List_of_tz_database_time_zones)
+*  `-r, --recurrence TEXT`      -  Recurrence on the scheduled execution. e.g. `2 weeks`, `30 min` or `1d`. You cannot use this argument with arguments: [rrule]
+*  `-c, --count INTEGER`        -  Maximum number of times to run the execution. If left empty, there's no limit on repetition. You cannot use this argument with arguments: [rrule]
+*  `--weekdays TEXT`            -  Weekdays on which to run the execution, e.g. `su,mo,tu`. If left empty, runs on any weekday. You cannot use this argument with arguments: [rrule]
+*  `--rrule TEXT`               -  A scheduling rule in the iCalendar format, e.g. `RRULE:FREQ=DAILY;INTERVAL=3`, which means run every 3 days. You cannot use this argument with arguments: [count, frequency, weekdays]
```

Comment:
"[count, frequency, weekdays]" -> drop parentheses. 

Holds for other places too.

Reply:
ditto